### PR TITLE
Release 19.0

### PR DESCRIPTION
**Pull request by jonparrott**

### CODE REVIEW

File: readme_renderer/__about__.py [hunk 1]
```diff
@@ -23,7 +23,7 @@
                'descriptions for Warehouse')
 __uri__ = "https://github.com/pypa/readme_renderer"
 
-__version__ = "18.1"
+__version__ = "19.0"
 
 __author__ = "Donald Stufft"
 __email__ = "donald@stufft.io"
```
Comment (brainwane) [inline readme_renderer/__about__.py]: We have a better email address we have started using for at least some of this stuff -- see https://github.com/pypa/warehouse/blob/master/warehouse/__about__.py .

File: CHANGES.rst [hunk 2]
```diff
@@ -1,6 +1,17 @@
 Changes
 =======
 
+19.0 (2018-04-10)
+-----------------
+
+* Allow details and summary tags (#77)
+* Add .pytest_cache to .gitignore. (#85)
```
Comment (brainwane) [inline CHANGES.rst]: Remove final period for consistency?

File: CHANGES.rst [hunk 3]
```diff
@@ -1,6 +1,17 @@
...
+* Warn about markdown content type when checking RST (#83)
+* Update pep8 tox env to use py3.6 (#84)
+* Add Pygments-based syntax highlighting for Markdown (#82)
+* Update docs about check to talk about Markdown (#80)
```
Comment (brainwane) [inline CHANGES.rst]: I'm being nitpicky here, but watch consistency in capitalization (e.g., Markdown versus markdown) in these.

File: readme_renderer/__about__.py (hunk 1 above)
Comment (jonparrott) [reply readme_renderer/__about__.py]: Great, done.

File: CHANGES.rst (hunk 2 above)
Comment (jonparrott) [reply CHANGES.rst]: Added periods to all of them to be more consistent with the rest of the changelog.

File: CHANGES.rst (hunk 3 above)
Comment (jonparrott) [reply CHANGES.rst]: Done.

File: readme_renderer/__about__.py [hunk 4]
```diff
@@ -23,10 +23,10 @@
                'descriptions for Warehouse')
 __uri__ = "https://github.com/pypa/readme_renderer"
 
-__version__ = "18.1"
+__version__ = "19.0"
 
-__author__ = "Donald Stufft"
-__email__ = "donald@stufft.io"
+__author__ = "The Python Packaging Authority"
+__email__ = "admin@mail.pypi.org"
```
Comment (dstufft) [inline readme_renderer/__about__.py]: This should probably distutils-sig or something.

Comment (brainwane) [reply readme_renderer/__about__.py]: I would like for it to be an address that is robustly monitored, and that doesn't require the user to sign up for something (such as a list) in order to contact the owner...

**Comment by jonparrott** [comment]
Published: https://pypi.org/project/readme_renderer/19.0/

**Pull request merged**